src/Resources/app/storefront/src/checkout/swag-paypal.acdc-fields.js

```diff
@@ -205,25 +206,26 @@ export default class SwagPaypalAcdcFields extends SwagPaypalAbstractStandalone {
             const firstInvalidFieldKey = Object.keys(state.fields).find((key) => !state.fields[key].isValid);
             this.fields[firstInvalidFieldKey]?.focus();
 
+            const field = DomAccess.querySelector(this.cardFieldForm, this.options[firstInvalidFieldKey + 'Selector']);
+
+            // field.scrollIntoView({
+            //     behavior: 'smooth',
+            //     block: 'center',
+            // });
```

Comment:
please no commented out code

Reply:
don't want it commented in as it could disturb with the `window.scrollTo` - do you have a better idea?

Reply:
Wait, so you want this to be commented in with v10? But right below you write `will be removed without replacement`. Then please comment better

Reply:
Oh yeah, that's my fault, I see where the confusion comes from - yes, that snippet is the replacement for `window.scrollTo`.